src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9541,6 +9541,7 @@ private bool ScanParenthesizedImplicitlyTypedLambda(Precedence precedence)
                 && (!this.IsInQuery || !IsTokenQueryContextualKeyword(this.PeekToken(1)))
                 && this.PeekToken(2).Kind == SyntaxKind.CommaToken)
             {
+                // Make sure it really looks like a lambda, not jut a tuple
```

Comment:
Thanks.
My keyboard has a problem with the 's' and the 'i'. Sometimes it will not hit and sometimes it will hit multiple times. Very annoying :-S